packages/twenty-front/src/modules/object-record/record-table/states/cursorByRecordFamilyState.ts

```diff
@@ -0,0 +1,9 @@
+import { createFamilyState } from '@/ui/utilities/state/utils/createFamilyState';
+
+export const cursorByRecordFamilyState = createFamilyState<
```

Comment:
I think we could remove this if we can rebuild the cursor from the recordId direclty. Would simplify it!